catalog/ACA/ACA_reef_habitat_v2_0.jsonnet

```diff
@@ -0,0 +1,320 @@
+      },
+      {
+        name: 'benthic',
+        description: 'Classification of dominant benthic composition.',
+        'gee:classes': [
+          {
+            color: '000000',
+            description: 'Unmapped',
+            value: 0,
+          },
+          {
+            value: 11,
+            color: 'ffffbe',
+            description: 'Sand - Sand is any soft-bottom area dominated by fine unconsolidated sediments.',
+          },
+          {
+            value: 12,
+            color: 'e0d05e',
+            description: 'Rubble - Rubble is any habitat featuring loose, rough fragments of broken reef material.',
+          },
+          {
+            value: 13,
+            color: 'b19c3a',
+            description: 'Rock - Rock is any exposed area of hard bare substrate.',
+          },
+          {
+            value: 14,
+            color: '668438',
+            description: 'Seagrass - Seagrass is any habitat where seagrass is the dominant biota.',
+          },
+          {
+            value: 15,
+            color: 'ff6161',
+            description: 'Coral/Algae - Coral/Algae is any hard-bottom area supporting living coral and/or algae.',
+          },
+          {
+            value: 18,
+            color: '9bcc4f',
+            description: |||
+              Microalgal Mats - Microalgal Mats are any visible accumulations of microscopic algae in sandy sediments.
+            |||,
+          },
+        ],
+      },
+      {
+        name: 'reef_mask',
+        description: |||
+          Globally standardised reef extent product that integrates a number of global 
+          reef classification and bahtymetry products ("Reef Extent" on the Atlas portal).
+        |||,
+        'gee:classes': [
+          {
+            color: '000000',
+            description: 'Not reef',
+            value: 0,
+          },
+          {
+            value: 1,
+            color: 'ffffff',
+            description: 'Reef',
+          },
+        ],
+      },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'Benthic Map',
+        lookat: {
+          lat: -17.00872,
+          lon: -149.56194,
+          zoom: 13,
+        },
+        image_visualization: {
+          band_vis: {
+            min: [
+              11.0,
+            ],
+            max: [
+              16.0,
+            ],
+            palette: [
+              'ffffbe',
+              'e0d05e',
+              'b19c3a',
+              '668438',
+              'ff6161',
+              '9bcc4f',
+            ],
+            bands: [
+              'benthic',
+            ],
+          },
+        },
+      },
+    ],
+  },
+  'sci:doi': '10.5281/zenodo.3833242',
+  'sci:citation': |||
+    Allen Coral Atlas (2020). Imagery, maps and monitoring of the world's tropical coral reefs. Zenodo. [doi:10.5281/zenodo.3833242](https://doi.org/10.5281/zenodo.3833242)
+  |||,
+  'gee:terms_of_use': ee.gee_terms_of_use(license),
+  'gee:user_uploaded': true,
```

Comment:
could you also add

'gee:skip_indexing': true

? It's a temporary thing we need to do for user-uploaded descriptions.